README.md

```diff
@@ -110,3 +110,61 @@ go run main.go discover --config source.json | jq .
   }
 }
 ```
+
+### 3. Read command:
+Create a `catalog.json` file that looks similar to the output of "Discover", but with the format of:
+```json
+{
+  "streams": [
+    {
+      "stream": { // body of stream from above },
+      "sync_mode": "full_refresh" // one of the supported_sync_modes from above
+    }
+  ]
+}
+```
+For the example output above, a valid `catalog.json` might look like:
+```json
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "departments",
+        "json_schema": {
+          "type": "object",
+          "properties": {
+            "dept_name": {
+              "type": "string"
+            },
+            "dept_no": {
+              "type": "string"
+            }
+          }
+        },
+        "supported_sync_modes": [
+          "full_refresh"
+        ],
+        "namespace": "planetscaledatabase"
+      },
+      "sync_mode": "full_refresh"
+    }
+  ]
+}
+```
+
+Then run:
+```bash
+go run main.go read --config source.json --catalog catalog.json
+```
+
+### 3a. Starting from a specific GTID
+You can start replication from a specific GTID _per keyspace shard_, by setting `starting_gtids` in your configuration file:
+```json
+{
+    "host": "<FQDN for your PS database>",
+    "database":"<default keyspace name>",
+    "username":"<username>",
+    "password":"<some password for your database>",
+    "starting_gtids": "{\"keyspace\": {\"shard\": \"MySQL56/MYGTID:1-3\"}}"
```

Comment:
Is there any reason we can't make this nicer JSON object instead of embedded JSON string? E.g. 

```
"starting_gtids": {
  "<keyspace>": {
    "<shard>": "<gtid>"
  }
}
```

Reply:
The only reason is that FiveTran does not support providing a JSON string field right now. These are the supported form fields: https://github.com/fivetran/fivetran_sdk/blob/main/common.proto#L18-L28; Airbyte will be the same 😄

Reply:
Most people interacting with the Airbyte connector will be doing so via these instructions (through the UI) 😄  https://planetscale.com/docs/integrations/airbyte

Making a note to update those docs after this PR is merged too.

Reply:
Also making a note to add tests to readSync, doesn't seem like that path has been tested.